src/app/shared/components/lmt-autocomplete/lmt-autocomplete.component.ts

```diff
@@ -99,6 +112,23 @@ export class LmtAutocompleteComponent {
     );
   }
 
+  onChange: any = () => {}
+
+  onTouch: any = () => {}
+  
+
+  writeValue(obj: any): void {
```

Comment:
public writeValue ( values: any[] ): void {
 this._selectedItems.push( ...values ) ;
}